lib/git-lfs-s3/version.rb

```diff
@@ -1,3 +1,3 @@
 module GitLfsS3
-  VERSION = '0.1.1'
+  VERSION = '0.2.2'
```

Comment:
Out of curiosity, how did you come up with the version change?  Also, I recommend against including this as part of a PR upstream.